sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -1,4 +1,17 @@
-WITH aic_flows AS (
+WITH flows_live AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_v1
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_live
+  WHERE
+    submission_date = CURRENT_DATE
```

Comment:
This will result in duplicates between 03:00 - midnight when `flows_v1` contains rows for the current date.  Either the `flows_v1` query should be filtered to before the current date, or the `flows_live` query should be filtered based on the max value from `flows_v1`.

Reply:
```suggestion
  SELECT
    *
  FROM
    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_v1
  WHERE
    submission_date < CURRENT_DATE
  UNION ALL
  SELECT
    *
  FROM
    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_live
  WHERE
    submission_date = CURRENT_DATE
```